### DISCUSSION TITLE
Phabricator Support

### DISCUSSION THREAD

**Discussion by raylenmargono** [concourse/concourse#6145] [Help & Support]
Hi,

I'm looking to use concourse as the remote build for phabricator. There isn't much support around this but was wondering if anyone has come up with a solution. 

Thank you!

**Comment by CodeMouse92** [comment]
I'm interested in this too. Basically, there has to be a way to trigger a build with HTTP POST, and Concourse needs to respond to that POST request with information on whether the build was successful or not.
